packages/react-composites/src/composites/CallWithChatComposite/state/CallWithChatAdapterState.ts

```diff
@@ -115,35 +115,28 @@ export interface CallWithChatAdapterState extends CallWithChatAdapterUiState, Ca
 /**
  * @private
  */
-export function callWithChatAdapterStateFromBackingStates(
-  callAdapter: CallAdapter,
-  chatAdapter: ChatAdapter
-): CallWithChatAdapterState {
+export function callWithChatAdapterStateFromBackingStates(callAdapter: CallAdapter): CallWithChatAdapterState {
   const callAdapterState = callAdapter.getState();
-  const chatAdapterState = chatAdapter.getState();
 
   return {
     call: callAdapterState.call,
-    chat: chatAdapterState.thread,
+    chat: undefined,
     userId: callAdapterState.userId,
     page: callAdapterState.page,
     displayName: callAdapterState.displayName,
     devices: callAdapterState.devices,
     isLocalPreviewMicrophoneEnabled: callAdapterState.isLocalPreviewMicrophoneEnabled,
     isTeamsCall: callAdapterState.isTeamsCall,
     latestCallErrors: callAdapterState.latestErrors,
-    latestChatErrors: chatAdapterState.latestErrors,
+    latestChatErrors: {},
     /* @conditional-compile-remove(file-sharing) */
-    fileUploads: chatAdapterState.fileUploads,
+    fileUploads: {},
```

Comment:
Can you please go in detail why these are set to empty objects and not chatAdapterState.<value>?
I see that chatAdapterState is available but how come we don't continue to utilize it here.